src/Chirp.Razor/Program.cs

```diff
@@ -1,7 +1,11 @@
 using Chirp.Razor.Pages;
+using CheepDB;
+using Repository;
 
 var builder = WebApplication.CreateBuilder(args);
 
+CheepRepository.InitDB();
+
```

Comment:
Just curious. What are the arguments for why it should by changed? :)

Reply:
The reaseon is that it's static and it's not supposed to be. When it gets changed back to a class we need to move it since it probably is not a good idea to have it there. The static method was just to see it work and dump all the data in the database :D

Reply:
Cool. Maybe change already then? if you commit and push to the same branch, it will automacally add it to this pull request.

Reply:
It has been changed

Reply:
Nice. Great